### PR TITLE
docs: update vector for greptimecloud and tls

### PR DESCRIPTION
**Pull request by sunng87**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [x] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by nicecui — APPROVED** [review]

File: docs/v0.7/zh/db-cloud-shared/clients/vector-integration.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
+Vector 是高性能的可观测数据管道。
```
Comment (tisonkun) [inline docs/v0.7/zh/db-cloud-shared/clients/vector-integration.md:1]:
```suggestion
[Vector](https://vector.dev) 是高性能的可观测数据管道。
```

**Review by tisonkun — APPROVED** [review]

**Comment by tisonkun** [comment]
Although I may prefer not to break line unless there is a logical line break.

**Comment by sunng87** [comment]
https://stackoverflow.com/questions/110928/is-there-a-valid-reason-for-enforcing-a-maximum-width-of-80-characters-in-a-code

**Comment by sunng87** [comment]
@nicecui remember to create a PR to update cloud console

**Comment by nicecui** [comment]
> @nicecui remember to create a PR to update cloud console

ref https://github.com/GreptimeTeam/greptime-cloud-frontend/pull/345

**Pull request merged**